app/src/main/java/com/naman14/hacktoberfest/fragment/ExploreFragment.java

```diff
@@ -271,7 +271,11 @@ public void onNext(List<Issue> response) {
                         swipeRefreshLayout.setEnabled(true);
                         loading = false;
                         if(page == 1)
-                            adapter.setData(response);
+                            if (response.isEmpty()) {
+                                Toast.makeText(getActivity(), "No matching projects found", Toast.LENGTH_SHORT).show();
```

Comment:
how about creating a variable in strings.xml and using it here?

Reply:
Yes that will make code reuseability easy